src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaClientFetchFactory.java

```diff
@@ -1113,6 +1121,14 @@ private int decodeFetchRecord(
                 final long timestampAbs = client.decodeRecordBatchTimestamp + recordHeader.timestampDelta();
                 final OctetsFW key = recordHeader.key();
 
+                client.decodeRecordOffset = offsetAbs;
+
+                if (KafkaConfiguration.DEBUG)
+                {
+                    System.out.format("[client] [0x%016x] %s[%d] FETCH Record %d %02x\n",
+                        client.replyId, client.topic, client.partitionId, client.decodeRecordOffset, recordHeader.attributes());
```

Comment:
Please remove the attributes from the debug output, they are unused at record level in the protocol.